packages/office-ui-fabric-react/src/components/TextField/TextField.scss

```diff
@@ -62,7 +62,8 @@
   font-size: $ms-font-size-m;
   color: $ms-color-neutralPrimary;
   height: 32px;
-  @include padding(6px, 12px, 7px, 12px);
+  line-height: 32px;
```

Comment:
Is this required? afaik this wil result in a cursor larger than the text in safari/ios